apps/frontend/src/components/global/HelpAboutDropdown.vue

```diff
@@ -40,16 +47,24 @@ import Vue from 'vue';
 import LinkItem from '@/components/global/sidebaritems/IconLinkItem.vue';
 import AboutModal from '@/components/global/AboutModal.vue';
 import HelpModal from '@/components/global/HelpModal.vue';
+import UserModal from '@/components/global/UserModal.vue';
+import {ServerModule} from '@/store/server';
+
 import Component from 'vue-class-component';
 
 @Component({
   components: {
     HelpModal,
     AboutModal,
+    UserModal,
     LinkItem
   }
 })
-export default class HelpAboutDropdown extends Vue {}
+export default class HelpAboutDropdown extends Vue {
+  get serverMode() {
+    return ServerModule.serverMode;
+  }
+}
```

Comment:
This can just take advantage of the [ServerMixin](https://github.com/mitre/heimdall2/blob/master/apps/frontend/src/mixins/ServerMixin.ts) instead of reimplementing the serverMode function.